controlsfx/src/main/java/org/controlsfx/control/textfield/AutoCompletionBinding.java

```diff
@@ -269,6 +269,27 @@ public final DoubleProperty maxWidthProperty() {
         return autoCompletionPopup.maxWidthProperty();
     }
 
+    /**
+     * Set the skin of the popup
+     */
+    public final void setSkin(Skin<?> skin) {
```

Comment:
What is the need for this? Is this PR linked to an issue? If not, maybe explain here or somewhere how this can be useful.

Reply:
I do not want to force users onto the default skin, they may have different requirements that are not matched by the current `AutoCompletePopupSkin` implementation. For example, if this method was public, I wouldn't have had to create a PR just because I needed a custom cell layout.